modules/instrument_list/php/instrument_list.class.inc

```diff
@@ -199,6 +199,7 @@ class Instrument_List extends \NDB_Menu_Filter
             $x            = -1;
             $prevSubgroup = null;
 
+            $i = 0;
```

Comment:
I don't think `$i` is being used for anything in this function.

Reply:
it's to prevent phan from flagging line 284 (`$i++`) as an error, because otherwise `$i` is initialized inside of an if statement and phan thinks it might be null